### ISSUE TITLE
CSS Styling not Working

### ISSUE THREAD

**Issue by jeff-bradley** [GeoNode/geonode#12716]
## Expected Behavior
Create a CSS style in geoserver, update the layer in geonode and have it display the style and then ability to edit css style in editor.
this behavior seemed to work in 4.1 #11434 

## Actual Behavior
Geoserver Fails to validate any CSS style, and CSS1.0.0 is not in supported_formats for geonode updatelayers

## Steps to Reproduce the Problem

  1. Create new CSS style in Geoserver
  2. Validate/Save the Style
  3. Page fails
 
**After removing asm-3.1.jar**
  1. Create new CSS style in Geoserver
  2. Save the Style
  3. attach style to layer
  5. run updatelayers for layer in django
  6. update gives warning of unsupported format

## Specifications

  - GeoNode version: 4.4
  - Installation type (vanilla, geonode-project): geonode-project
  - Installation method (manual, docker): docker
  - Platform: Ubuntu 20
  - Additional details:
 I was able to get the desired output by first removing the asm-3.1.jar file from WEB-INF/lib and restarting Geoserver, This would solve the Geoserver issue but then when I would use updatelayers for that layer in geonode i would get this error

```
Inspecting the available layers in GeoServer ...
Found 1 layers, starting processing

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/geoserver/catalog.py", line 1237, in get_style
    return Style(
  File "/usr/local/lib/python3.10/dist-packages/geoserver/style.py", line 34, in __init__
    assert style_format in Style.supported_formats
AssertionError
```
I then added a new css1.0.0 value to the /geoserver/style.py 
```
    supported_formats = ["sld10", "sld11", "zip10", "css10", "sld1.0.0", "css1.0.0"]
    content_types = {
        "sld10": "application/vnd.ogc.sld+xml",
        "sld11": "application/vnd.ogc.se+xml",
        "zip10": "application/zip",
        "css10": "application/vnd.geoserver.geocss+css",
        "css1.0.0": "application/vnd.geoserver.geocss+css",
        "sld1.0.0": "application/vnd.ogc.sld+xml",
    }
```
this allowed me to successfully update the layer and use CSS styling

**Comment by etj** [comment]
Trying a fix with https://github.com/geoserver/geoserver/pull/8028

**Comment by giohappy** [comment]
@jeff-bradley the fix has been backported to GeoNode 4.4.x branch, where Geoserver 2.24 SNAPSHOT is configured.
